packages/lib/src/logger.ts

```diff
@@ -5,6 +5,10 @@ import { configure, Configuration as LoggingConfig, Logger, Level } from 'log4js
 import { Logger as APILogger } from '@smartthings/core-sdk'
 
 
+const DEFAULT_LOG_FILE_SIZE = 1048576 // bytes
```

Comment:
Super trivial but you could make this is little more readable by using something like `1024 * 1024`, or, since we probably don't need a full MiB, we could use a MB and use underscores since it's so long: `1_000_000`.

Reply:
I like the numeric separator.